Feature/106/v2.2.0 docs (#107)
sakuli/sakuli.github.io: content/docs/enterprise/_index.md

Some features of Sakuli require an enterprise license. Please consult our **[overview](/enterprise)** to see and request packages and prices. After you registered for an enterprise subscription, you will get a license-key and an NPM access token which are required to use enterprise features.

## Assisted Setup

It is assumed that you have setup your environment as described in the [Getting Started Guide](/docs/getting-started).

> On the command line navigate to the root folder of your Sakuli project (where the `package.json` file is located) and run the following command:

Reviewer: I think there is a word missing?

{{<highlight bash>}}
npx sakuli enable-enterprise
{{</highlight>}}

> This command starts an assistant which will guide you through the setup of enterprise features. The first question is regarding the Sakuli-license key which you should have obtained when you have a enterprise subscription.

Reviewer: The first question is regarding the Sakuli-license key which you should have received after subscribing to an enterprise package.

{{<highlight bash>}}
? Would you like to enter your Sakuli license information? (Y/n)
{{</highlight>}}

> You can enter `n` when you already entered license information before.

Reviewer 2: ```suggestion
You can enter `n` when you have already entered the license information before.
```

> You can enter `Y` and than enter / paste the license key in the next prompt:

Reviewer 2: ```suggestion
You can enter `Y` and paste the license key in the next prompt:
```

{{<highlight bash>}}
? Please enter your license key:
{{</highlight>}}

Same applies to the NPM access token which can be pasted / entered in the next prompt:

{{<highlight bash>}}
? Please enter your npm key:
{{</highlight>}}

> The upcoming prompt asks you about the enterprise features which you want to activate. You can use <kbd>↑</kbd> and <kbd>↓</kbd> to navigate the cursor and <kbd>SPACE</kbd> select or deselect a feature. Pressing <kbd>ENTER</kbd> will submit your selection and starts the setup process.

Reviewer: *to select or deselect a feature. .... and start the

> The upcoming prompt asks you about the enterprise features which you want to activate. You can use <kbd>↑</kbd> and <kbd>↓</kbd> to navigate the cursor and <kbd>SPACE</kbd> select or deselect a feature. Pressing <kbd>ENTER</kbd> will submit your selection and starts the setup process.

Reviewer 2: ```suggestion
The following prompt asks you about the enterprise features which you want to activate. You can use <kbd>↑</kbd> and <kbd>↓</kbd> to navigate the cursor and <kbd>SPACE</kbd>  to select or deselect a feature. Pressing <kbd>ENTER</kbd> will submit your selection and start the setup process.
```

{{<highlight bash>}}
? Please Select Enterprise Features to bootstrap (Press <space> to select, <a> to toggle all, <i> to invert selection)
❯◯ Forwarding to Icinga2
 ◯ Forwarding to Check_MK
 ◯ Forwarding to OMD
{{</highlight>}}

> Sakuli will than configure the license information globally and add instal and preconfigure the selected enterprise features in your project.

Reviewer: will th**e**n configure ... globally and instal**l** and preconfigure

> The next sections is show how to setup enterprise features manually and in more detail. You can now just start using your feature:

Reviewer: is about how
